go/logger/context_with_logging.go

```diff
@@ -0,0 +1,58 @@
+package logger
+
+import (
+	"testing"
+
+	"golang.org/x/net/context"
+)
+
+type ContextAndLogger struct {
```

Comment:
is `logger.Context` ok?

Reply:
How about `logger.MiniContext` ?

Reply:
Not a big fan of `ContextAndLogger` since once we add a 3rd field, we have to change everything.

Reply:
I still want a better name for this ^

Reply:
I'll go for logger.Context. It is weird to me to use MiniContext for something that is bigger than a context.